app/src/test/java/com/kickstarter/viewmodels/BackingFragmentViewModelTest.kt

```diff
@@ -94,6 +96,7 @@ class BackingFragmentViewModelTest : KSRobolectricTestCase() {
         this.vm.outputs.projectDataAndAddOns().subscribe(this.listAddOns)
         this.vm.outputs.bonusSupport().map { it.toString() }.subscribe(this.bonusAmount)
         this.vm.outputs.deliveryDisclaimerSectionIsGone().subscribe(this.disclaimerSectionIsGone)
+        Shadows.shadowOf(Looper.getMainLooper()).idle()
```

Comment:
I think these `Shadows.shadowOf(Looper.getMainLooper()).idle()` are no longer necessary as you found the root cause if blocking problem

Reply:
Done, and everything passed CI! 🎉 🚀